dags/resources/stages/parse/table_definitions/arbitrum/SequencerInbox_call_addSequencerL2BatchFromOrigin.json

```diff
@@ -0,0 +1,82 @@
+{
+    "parser": {
+        "abi": {
+            "inputs": [
+                {
+                    "internalType": "uint256",
+                    "name": "sequenceNumber",
+                    "type": "uint256"
+                },
+                {
+                    "internalType": "bytes",
+                    "name": "data",
+                    "type": "bytes"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "afterDelayedMessagesRead",
+                    "type": "uint256"
+                },
+                {
+                    "internalType": "contract IGasRefunder",
+                    "name": "gasRefunder",
+                    "type": "address"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "prevMessageCount",
+                    "type": "uint256"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "newMessageCount",
+                    "type": "uint256"
+                }
+            ],
+            "name": "addSequencerL2BatchFromOrigin",
+            "outputs": [],
+            "stateMutability": "nonpayable",
+            "type": "function"
+        },
+        "contract_address": "0xd03bfe2ce83632f4e618a97299cc91b1335bb2d9",
+        "field_mapping": {},
+        "type": "trace"
+    },
+    "table": {
+        "dataset_name": "<INSERT_DATASET_NAME>",
+        "schema": [
+            {
+                "description": "",
+                "name": "sequenceNumber",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "data",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "afterDelayedMessagesRead",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "gasRefunder",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "prevMessageCount",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "newMessageCount",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "<TABLE_PREFIX>_event_addSequencerL2BatchFromOrigin"
```

Comment:
Should be `SequencerInbox_call_addSequencerL2BatchFromOrigin` instead of `<TABLE_PREFIX>_event_addSequencerL2BatchFromOrigin`